app/src/ingest_policy_pdfs.py

```diff
@@ -22,11 +32,93 @@ def _ingest_policy_pdfs(
     embedding_model = app_config.sentence_transformer
     for file in file_list:
         if file.endswith(".pdf"):
+            with smart_open_file(file, "rb") as fin:
+                extract_text_to_fp(fin, output_string, laparams=LAParams(), output_type="text")
+
+                parse_pdf_and_add_to_db(
+                    contents=output_string.getvalue(),
+                    db_session=db_session,
+                    doc_attribs=doc_attribs,
+                    embedding_model=embedding_model,
+                )
+
             logger.info(
                 f"Processing pdf file: {file} at {pdf_file_dir} using {embedding_model}, {db_session}, with {doc_attribs}"
             )
 
 
+def parse_pdf_and_add_to_db(
+    contents: str,
+    db_session: db.Session,
+    doc_attribs: dict[str, str],
+    embedding_model: SentenceTransformer,
```

Comment:
Instead of passing db_session and embedding_model as parameters, grab them from `app_config`